pantheon/src/main/java/tech/pegasys/pantheon/cli/util/TomlConfigFileDefaultProvider.java

```diff
@@ -140,4 +143,22 @@ private void loadConfigurationFromFile() {
 
     checkConfigurationValidity();
   }
+
+  private void checkUnknownOptions(final TomlParseResult result) {
+    final CommandLine.Model.CommandSpec commandSpec = commandLine.getCommandSpec();
```

Comment:
import CommandLine.Model.CommandSpec so it doesn't need namespacing here please

Reply:
👍 fixed.